.github/workflows/build.yml

```diff
@@ -0,0 +1,24 @@
+name: Build NetKAN-Infra
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
```

Comment:
I thought QEMU was a machine emulator like VMware. Why do we need it to build NetKAN-Infra?

Reply:
It is included in the usage examples for the Docker build-push action:
See https://github.com/marketplace/actions/build-and-push-docker-images#usage

Not sure if it's actually a hard dependency for this buildx or not. They talk about it being used for "multi-platform" images in the [README](https://github.com/docker/buildx#building-multi-platform-images).

Reply:
OK, those links make it sound like we don't need QEMU...

- We only need to build for one arch (whatever we're using for the bot)
- It's just Python anyway
- We don't need to run binaries from other architectures

Is there a way I can test this locally with those lines removed, to confirm?

Reply:
Yeah, fair points. I'll try it, the documentation wasn't super clear about the point.

Reply:
Seems to work, I'll remove all the QEMU statements. Saves us 10 seconds on each build :smile: